front/app/modules/commercial/insights/hooks/useInsightsInputs.ts

```diff
@@ -37,13 +38,15 @@ const useInsightsInputs = (
   const category = queryParameters?.category;
   const search = queryParameters?.search;
   const sort = queryParameters?.sort;
+  const processed = queryParameters?.processed;
```

Comment:
Wondering if this shouldn't default as true, but no strong opinion there

Reply:
I would leave it as is, passing true/false to this hook is already handled by using the `getInputsCategoryFilter` util.